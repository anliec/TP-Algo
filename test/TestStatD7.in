STATS_D7 1

ADD 2 2015 06 23 20 26 1 V
STATS_D7 1
ADD 2 2015 06 23 20 25 1 J
STATS_D7 1
ADD 2 2015 06 23 20 24 1 R
STATS_D7 1
ADD 2 2015 06 23 20 23 1 N
STATS_D7 1

ADD 3 2015 06 23 20 26 1 V
STATS_D7 1

STATS_D7 2
EXIT
