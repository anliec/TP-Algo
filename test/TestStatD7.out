V 0%
J 0%
R 0%
N 100%
V 100%
J 0%
R 0%
N 0%
V 50%
J 50%
R 0%
N 0%
V 33%
J 33%
R 33%
N 0%
V 25%
J 25%
R 25%
N 25%
V 40%
J 20%
R 20%
N 20%
V 0%
J 0%
R 0%
N 100%
